it/mail/chrome/messenger/am-dsn.dtd

```diff
@@ -1,16 +1,16 @@
 <!ENTITY pane.title                 "Notifica dello stato di consegna">
-<!ENTITY useGlobalPrefs.label       "Use my global Delivery Status Notification preferences for this account">
+<!ENTITY useGlobalPrefs.label       "Utilizza le mie preferenze globali per la Notifica dello stato di consegna per questo account">
 <!ENTITY useGlobalPrefs.accesskey   "U">
-<!ENTITY useCustomPrefs.label       "Customize Delivery Status Notification for this account">
+<!ENTITY useCustomPrefs.label       "Personalizza la Notifica dello stato di consegna per questo account">
 <!ENTITY useCustomPrefs.accesskey   "C">
-<!ENTITY requestAlways.label        "When sending messages, always request a Delivery Status Notification">
+<!ENTITY requestAlways.label        "Durante l'invio dei messaggi, richiedi sempre una Notifica dello stato di consegna">
```

Comment:
CORRECTIONS NEEDED. 

The main clause should always be placed before the subordinate clause. The correct entry should be "Richiedi sempre una Notifica dello stato di consegna durante l'invio dei messaggi".